backend/settings.js

```diff
@@ -90,6 +88,18 @@ function getDerivedSetting(settingName) {
                 'settings.yaml'
             );
 
+        case 'KEY_FILE':
+            return path.join(
+                getSetting('STORAGE_PATH'),
+                'privkey.pem'
+            );
+
+        case 'CERT_FILE':
+            return path.join(
+                getSetting('STORAGE_PATH'),
+                'fullchain.pem'
```

Comment:
Same as above - for the names and extensions I kept it consistent with what letsencrypt generates automatically.